lib/templates/text.mustache

```diff
@@ -1,4 +1,4 @@
-<div class="res-text usertext-body">
+<div class="res-text usertext-body res-expando-host-{{moduleId}}">
```

Comment:
Good thought + makes it a little more generic so it can work for all mediahosts regardless of type 👍